src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -135,7 +135,13 @@ <h3 *ngIf="displaySubsection(showQuestion, 0, 4)">Setting Up Sessions</h3>
           <p>
             This is the form used to set up sessions.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <tm-session-edit-form [formMode]="SessionEditFormMode.ADD"
```

Comment:
I think somehow you may also want to add a `isDisplayOnly` or `disabled` field to the component unless you want the form to be interactive (which is also acceptable as the event thrown by the component is ignored).

Reply:
The form is made not interactive with the `isEditable` field in the model so I did not feel the need to add an additional disabled field to the components